README.md

```diff
@@ -6,30 +6,12 @@ _Note: once the connector is published, it will be available to everyone in the
 
```

Comment:
change "and you can skip to the ..." to "and you can use it without building or creating a custom connector"